### PR TITLE
nate/fix/LOOP-1814/food-type-absorption

### PR DESCRIPTION
**Pull request by nhamming**
https://tidepool.atlassian.net/browse/LOOP-1814

first entry custom food type adjust absorption time

### CODE REVIEW

File: Loop/View Controllers/CarbEntryViewController.swift [hunk 1]
```diff
@@ -493,11 +495,13 @@ extension CarbEntryViewController: EmojiInputControllerDelegate {
             return
         }
 
-        let lastAbsorptionTime = self.absorptionTime
-        self.absorptionTime = orderedAbsorptionTimes[section]
-
-        if let cell = tableView.cellForRow(at: IndexPath(row: Row.absorptionTime.rawValue, section: 0)) as? DateAndDurationTableViewCell {
-            cell.duration = max(lastAbsorptionTime ?? 0, orderedAbsorptionTimes[section])
+        if foodType == nil || foodType!.isEmpty {
```
Comment (darinkrauss) [inline Loop/View Controllers/CarbEntryViewController.swift]: What will happen if the user taps a fast absorption emoji, then deletes it, then taps a slow absorption emoji? This logic will change it for both emoji. Is that desired here?

**Review by darinkrauss — APPROVED** [review]
One comment, but assuming this is the desired behavior, LGTM! ☘️

File: Loop/View Controllers/CarbEntryViewController.swift (hunk 1 above)
Comment (nhamming) [reply Loop/View Controllers/CarbEntryViewController.swift]: Correct. I understand that the desired behaviour is that only the first entered emoji updates the absorption time.

**Review by darinkrauss — APPROVED** [review]
LGTM! ⚽

**Pull request merged**